components/cluster/pcs/Makefile

```diff
@@ -26,6 +27,87 @@ COMPONENT_ARCHIVE_URL= https://github.com/ClusterLabs/$(COMPONENT_NAME)/archive/
 COMPONENT_LICENSE= GPLv2
 COMPONENT_SUMMARY= Pacemaker command line interface and GUI.
 
+#
+# The build process tries to use wget to download an archive without even a
+# check for the correct hash.  In order to allow the build to work without
+# networking, we download it first.  This will need to be updated if the
+# version or URL in "pcs/snmp/Makefile" changes.
+#
+PYAGENTX_VERSION= 0.4.pcs.1
+COMPONENT_ARCHIVE_1= pyagentx-v$(PYAGENTX_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH_1= sha256:700ee61aef73f4c7af6dcf368c44d84479227112aaa535651a09665484aa30cc
+COMPONENT_ARCHIVE_URL_1= https://github.com/ondrejmular/pyagentx/archive/v$(PYAGENTX_VERSION).tar.gz
+
+#
+# The package includes a private copy of a series of Ruby Gems.  We must
+# download these in advance so that the build can run without network access.
+#
+COMPONENT_ARCHIVE_2= backports-3.11.3.gem
+COMPONENT_ARCHIVE_HASH_2= sha256:57b04d4e2806c199bff3663d810db25e019cf88c42cacc0edbb36d3038d6a5ab
+COMPONENT_ARCHIVE_URL_2= https://rubygems.org/gems/backports-3.11.3.gem
+CACHED_GEM_FILES += $(COMPONENT_ARCHIVE_2)
```

Comment:
You should use $(COMPONENT_ARCHIVE_2) here instead repeating the name.

Reply:
Good point!  I have made this change for all the gem files.